scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
I'm +1 on removing gox. Let's do that. `parallel` I'm not sure why that would be an issue.

@cpuguy83 can you tell me more about your development workflow? Are you building in containers or doing things on your desktop/laptop?

If it's just for testing, there's no reason that `GOARCH=... go build cmd/docker/` wouldn't work.